api/src/main/java/org/pmiops/workbench/workspaces/WorkspacesController.java

```diff
@@ -425,15 +379,22 @@ private void setDbWorkspaceFields(
       dbWorkspace.setReviewRequested(researchPurpose.getReviewRequested());
     }
 
-    updateWorkspaceBillingAccount(dbWorkspace, request.getWorkspace().getBillingAccountName());
+    try {
+      workspaceService.updateWorkspaceBillingAccount(
```

Comment:
The PATCH API is only supposed to include fields that have actually been modified. See all the other conditional checks above. This should mirror that behvaior